pydra/engine/core.py

```diff
@@ -531,7 +537,8 @@ def nodes(self):
 
     @property
     def graph_sorted(self):
-        return list(nx.topological_sort(self.graph))
+        self._graph_sorted[:] = list(nx.topological_sort(self.graph))
+        return self._graph_sorted
```

Comment:
keep this the original property

Reply:
I had to do this way, because didn't work properly without the additional assignment, had to be sure that it doesn't create a new object. Right now I can't recall details, can check later